apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/ContextManagerExtendService.java

```diff
@@ -61,7 +63,7 @@ public AbstractTracerContext createTraceContext(String operationName, boolean fo
         }
 
         int suffixIdx = operationName.lastIndexOf(".");
-        if (suffixIdx > -1 && Config.Agent.IGNORE_SUFFIX.contains(operationName.substring(suffixIdx))) {
+        if (suffixIdx > -1 && Arrays.stream(Config.Agent.IGNORE_SUFFIX.split(",")).anyMatch(a -> a.equals(operationName.substring(suffixIdx)))) {
```

Comment:
The `split` should happen in the `boot` stage, and only do match in the runtime.